### ISSUE TITLE
revivification

### ISSUE THREAD

**Issue by yamins81** [neuroailab/tfutils#28]
We should have the capability of "revivifying" saved data to allow from continuing training/testing without having to pass parameters (other than the load params). 

This should be basically possible because the saved data contains all the parameters that allowed for the model, training data, loss, optimizer, etc, to be constructed in the first place.   Functions and modules can be imported for the modname/objname info.    The only problem is potentially that numpy arrays and tuples won't be recoverable in their exact form (since JSON doesn't distinguish these from lists).
